src/app/services/country.service.ts

```diff
@@ -0,0 +1,18 @@
+import { HttpHeaders } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CountryService {
+  country: string = 'us';
+  constructor() {}
+
+  setService(country: string) {
+    this.country = country;
+  }
+
+  sendHeader():HttpHeaders {
```

Comment:
`sendHeader` is a missleading method name, this should be called something like `getHttpHeaders`

Reply:
changed sendHeader to hetHttpHeaders